### PR TITLE
Support PHP8.0

### PR DESCRIPTION
**Pull request by godruoyi**
Hi guys, this PR wants to upgrade the latest dependency version to php8.0, since php8.2 is coming soon, may it's time to upgrade it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,12 @@ Add the dependency to `phpmentors/workflower` into your `composer.json` file as
 composer require phpmentors/workflower "1.4.*"
 ```
 
+when your PHP version >= 8.0, you can install `1.5.*`
+
+```
+composer require phpmentors/workflower "1.5.*"
+```
+
```
Comment (godruoyi) [inline README.md:65]: Need to publish a new version with `v1.5.0`.

**Comment by iteman** [comment]
@godruoyi Thank you!

The `master` branch is currently under development. Could you change this PR for `v.1.4` branch?

**Comment by godruoyi** [comment]
Hi, @iteman, Do you have any plan to publish the master branch code?

**Comment by godruoyi** [comment]
@iteman move to https://github.com/phpmentors-jp/workflower/pull/92, PTAL.

**Pull request closed without merging**